src/nifs.sol

```diff
@@ -0,0 +1,335 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import "src/pasta/Pallas.sol";
+import "src/pasta/Vesta.sol";
+import "src/verifier/step2/Step2Data.sol";
+import "src/poseidon/Sponge.sol";
+
+library NIFSPallas {
+    uint256 constant private MOD = Pallas.P_MOD;
```

Comment:
Forgotten constant? This isn't defined in Pallas.

Reply:
The constants should be defined [here](https://github.com/lurk-lab/solidity-verifier/blob/419913d22b333238f57868fcd6278ea7be04e0e2/src/pasta/Pallas.sol#L21). The output of `forge test` having a single failing test should be enough to verify that the constant is being seen, but maybe your IDE is unable to resolve the dependencies? (I recall having to fiddle with the suggestions here to get my VS Code setup working: https://book.getfoundry.sh/config/vscode#1-remappings)

Reply:
Ah, apologies, you're correct! Thanks for the pointer!